### PR TITLE
docs(examples): modernize website examples.md

### PR DESCRIPTION
**Pull request by schroederc**

### CODE REVIEW

File: kythe/web/site/examples.md [hunk 1]
```diff
@@ -122,17 +115,18 @@ java -Xbootclasspath/p:third_party/javac/javac*.jar \
...
 #   indexed compilation
 
-# Produce the .kindex files for each compilation in the Kythe repo
+# Produce the .kzip files for each compilation in the Kythe repo
 ./kythe/extractors/bazel/extract.sh "$PWD" .kythe_compilations
 
 # Index the compilations, producing a GraphStore containing a Kythe index
-docker run --rm \
+bazel build //kythe/release:docker
+docker run --rm -ti \
```
Comment (justbuchanan) [inline kythe/web/site/examples.md]: Is the `-ti` flag needed? I've only used that before when running something interactive in the container.

File: kythe/release/kythe.sh [hunk 2]
```diff
@@ -29,9 +29,7 @@ example: docker run --rm -t -v "$HOME/repo:/repo" -v "$HOME/gs:/graphstore" \
 Extraction:
```
Comment (justbuchanan) [inline kythe/release/kythe.sh]: The `[--index_pack]` in the usage above can be deleted.

**Comment by justbuchanan** [comment]
One more minor fix: The command that says to run `bazel run bazel run //kythe/go/storage/tools:triples` should be changed to `//kythe/go/storage/tools/triples` (slash instead of colon).

**Comment by schroederc** [comment]
PTAL

**Pull request merged**